setup.py

```diff
@@ -43,6 +41,9 @@
             'coverage',
             'pytest >= 2.6.0',
             'pytest-cov',
-            'webtest'
+            'webtest',
+            'tox',
+            'flake8'
```

Comment:
As mentioned in the other project, adding flake8 should not be necessary as it gets installed via tox (deps=...).

Installing tox is okay, if that helps you developing (as mentioned in the other repo, I have installed tox globally as I need it for every Python project).

Reply:
Done